### PR TITLE
doc using kubectl, update samples

### PR DESCRIPTION
**Pull request by oneilcin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,16 +63,46 @@ helm install --name cma-ssh deployments/helm/cma-ssh/
...
 
-## Creating a kubernetes cluster with cma-ssh
+## Creating kubernetes clusters with cma-ssh using kubectl
 
-Using kubectl, apply a cluster manifest, and one or more machine manifests to create a kubernetes cluster.
+Either kubectl or the Swagger UI REST interface can be used to create kubernetes clusters with cma-ssh.  This section will focus on using kubectl.
 
-- [cluster spec](https://github.com/samsung-cnct/cma-ssh/blob/master/samples/cluster/cluster_v1alpha1_cluster.yaml) contains the kubernetes version.
--  [machine spec](https://github.com/samsung-cnct/cma-ssh/blob/master/samples/cluster/cluster_v1alpha1_machine.yaml) contains a role, label, and instanceType.
+A cluster definition consists of two kinds of kubernetes customer resource definitions (CRDs):
```
Comment (davidewatson) [inline README.md]:
```suggestion
A cluster definition consists of two kinds of Kubernetes Custom Resource Definitions (CRDs):
```

File: README.md [hunk 2]
```diff
@@ -63,16 +63,46 @@ helm install --name cma-ssh deployments/helm/cma-ssh/
...
+- [cnctcluster CRD](https://github.com/samsung-cnct/cma-ssh/blob/master/crd/cluster_v1alpha1_cnctcluster.yaml), and 
+- [cnctmachine CRD](https://github.com/samsung-cnct/cma-ssh/blob/master/crd/cluster_v1alpha1_cnctmachine.yaml)
+
+A single cluster definition consists of:
+- one [cnctcluster resource](https://github.com/samsung-cnct/cma-ssh/blob/master/samples/cluster/cluster_v1alpha1_cluster.yaml), and 
+- one or more [cnctmachine resources](https://github.com/samsung-cnct/cma-ssh/blob/master/samples/cluster/cluster_v1alpha1_machine.yaml) to define master and worker nodes.
+
+### Namespace per cluster
+
+The resources for a single cluster definition must be in the same namespace.  You cannot define two clusters in the same namespace, each cluster requires it's own namespace.  One naming option is to use unique cluster names and define a namespace that matches the cluster name.
```
Comment (davidewatson) [inline README.md]: Can you please make these lines less than approximately 80 characters? It is hard to review changes when there are so many words on the same line.

File: README.md [hunk 3]
```diff
@@ -63,16 +63,46 @@ helm install --name cma-ssh deployments/helm/cma-ssh/
...
 
-## Creating a kubernetes cluster with cma-ssh
+## Creating kubernetes clusters with cma-ssh using kubectl
 
-Using kubectl, apply a cluster manifest, and one or more machine manifests to create a kubernetes cluster.
+Either kubectl or the Swagger UI REST interface can be used to create kubernetes clusters with cma-ssh.  This section will focus on using kubectl.
 
-- [cluster spec](https://github.com/samsung-cnct/cma-ssh/blob/master/samples/cluster/cluster_v1alpha1_cluster.yaml) contains the kubernetes version.
--  [machine spec](https://github.com/samsung-cnct/cma-ssh/blob/master/samples/cluster/cluster_v1alpha1_machine.yaml) contains a role, label, and instanceType.
+A cluster definition consists of two kinds of kubernetes custom resource definitions (CRDs):
```
Comment (davidewatson) [inline README.md]:
```suggestion
A cluster definition consists of two kinds of Kubernetes Custom Resource Definitions (CRDs):
```

File: README.md (hunk 2 above)
Comment (oneilcin) [reply README.md]: yes

File: README.md (hunk 3 above)
Comment (oneilcin) [reply README.md]: done

**Pull request merged**